Dockerfile.agnos

```diff
@@ -83,28 +80,34 @@ RUN cd /tmp && apt-get -o Dpkg::Options::="--force-overwrite" install -yq ./capn
 COPY --from=agnos-compiler-ffmpeg /tmp/ffmpeg.deb /tmp/ffmpeg.deb
 RUN cd /tmp && apt-get -o Dpkg::Options::="--force-overwrite" install -yq ./ffmpeg.deb
 
-# Use pre-compiled PyQt5 wheel
-COPY --from=agnos-compiler-pyqt5 /tmp/PyQt5-5.15.9/PyQt5-5.15.9-cp38-abi3-manylinux_2_17_aarch64.whl /tmp/PyQt5-5.15.9-cp38-abi3-manylinux_2_17_aarch64.whl
+ENV XDG_DATA_HOME="/usr/local"
+ENV PATH="$XDG_DATA_HOME/.cargo/bin:$PATH"
+ARG rebuildpyqt5=0
```

Comment:
any reason not to do this every time now that it's parallel?

Reply:
Considering the todo item PyQt5: either need a script to build/pull the wheel or build/install in the Docker build again,
Even with parallel builds, seems building wheel every time would slow down the build. I have put build flag rebuildpyqt5=0 when set to 0 uses predefined wheel when set to 1 builds the wheel. I suppose other options could include to stay with building the wheel every time but then using prebuilt wheel becomes redundant.

Reply:
How did you build the wheel? rebuildpyqt5 does not actually rebuild it as in saving an updated whl file, but just chooses between your file and building from scratch.

Reply:
@robin-reckmann rebuildpyqt5 would either select the prebuilt wheel or build again from source and then save the file for use. Perhaps the better alternative as suggested by @adeebshihadeh is to rebuild wheel every time but doing so makes the prebuilt wheel redundant so perhaps drop the prebuilt wheel and rebuild wheel every time.

Reply:
@robin-reckmann rebuildpyqt5 would either select the prebuilt wheel or build again from source and then save the file for use. Perhaps the better alternative as suggested by @adeebshihadeh is to rebuild wheel every time but doing so makes the prebuilt wheel redundant so perhaps remove the prebuilt wheel and rebuild wheel every time.

Reply:
Yes, ideally we would just use the whl file and have a completely separated script to build it on demand. My only issue with your solution is, that the whl file is not actually saved outside of the docker build context. And the ARG is not expose to the build script.

Reply:
@adeebshihadeh If the .whl file were available on the PyPI server, we probably wouldn't have an issue using it, or?(Similar to not re-building all the deb files). Given that we don't even modify it (for now) and it saves a considerable amount of build time, I would again propose to completely separate out the whl compile and use that to update if necessary.

Reply:
Let's build it every time. I already explained my rationale in the other PR